### PR TITLE
[FIX] [15.0] sale_quotation_builder: fix translation for `website_description` field

### PR DESCRIPTION
**Pull request by vietphuong97**
Current behavior before PR:
- With current logic, all translations of field `website_description` will follow the translation in the client's language.
- This won't be a problem if the client doesn't change their language outside of the portal page.
- When they change to another language, the Sale Order information will get mixed up in multiple languages, which will annoy them.

Solution:
- set compute store=false for `website_description` field, each time this translation is viewed, it will be based on the language in the environment.
- This I find not affected much in terms of design

Desired behavior after PR is merged: Information of Sell Orders will always be in the same language.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by vietphuong10** [comment]
hi @thiryr , can you ping the security team or review this PR?

**Comment by thiryr** [comment]
Change seems to make sense to me but it's out of my scope :)

Team Website is already pinged on this. Try pinging them again in a while if they don't review it.
This doesn't seem to need an advanced review from security since we don't change anything, so no need to ping them explicitly for now.

**Comment by vietphuong10** [comment]
@rd-

> Change seems to make sense to me but it's out of my scope :)
> 
> Team Website is already pinged on this. Try pinging them again in a while if they don't review it. This doesn't seem to need an advanced review from security since we don't change anything, so no need to ping them explicitly for now.

Thank you. I will wait.

**Comment by rdeodoo** [comment]
> Team Website is already pinged on this. 

Hello, it's not in our scope, it's related to sale/ecommerce @odoo/rd-ecommerce , especially since they refactored those fields in Odoo 16, they will have a better idea about it :+1:

File: addons/sale_quotation_builder/models/product_template.py
Comment (Feyensv) [inline addons/sale_quotation_builder/models/product_template.py:20]: AFAIK, this doesn't change anything since you're putting as lang the existing environment language.

**Review by Feyensv — CHANGES_REQUESTED** [review]
Such a change is not acceptable in stable.

First of all, converting fields to compute is a big change that should be avoided. 
Second, storing (or not storing anymore) a field is a database change, thus not acceptable in stable either.

The `website_description` may not be properly translated in all cases, but I believe it could be fixed with the proper context here and there, not with such a heavy approach in stable.

**Comment by vietphuong10** [comment]
> Such a change is not acceptable in stable.
> 
> First of all, converting fields to compute is a big change that should be avoided. Second, storing (or not storing anymore) a field is a database change, thus not acceptable in stable either.
> 
> The `website_description` may not be properly translated in all cases, but I believe it could be fixed with the proper context here and there, not with such a heavy approach in stable.

I found that, to fix the current problem, it would be necessary to fix most of the models logic that appears in this module (tried with onchange fix).

Then I checked that this field, in modules and addons, it has almost no effect on the current flow of activity if switched to compute (store=False).

I haven't found a simpler solution yet. Can you come up with some other solutions?

File: addons/sale_quotation_builder/models/product_template.py
Comment (vietphuong10) [reply addons/sale_quotation_builder/models/product_template.py:20]: > AFAIK, this doesn't change anything since you're putting as lang the existing environment language.

yes, changing this place seems a bit redundant

**Pull request closed without merging**
